hack/start

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
d: Why did you put this script in the hack folder? I'd prefer to have this in the root folder of the repo, because i would not search for a start script in the hack folder. And it doesn't look like a hack to me but just a helpful start script ;)

Reply:
idk, it got kind of conventional in the k8s space to have a hack folder with helper scripts etc. 
I don't mind either way